letsencrypt/client.py

```diff
@@ -125,6 +130,21 @@ def register(config, account_storage, tos_cb=None):
 
     return acc, acme
 
+def perform_registration(acme, config):
+    """
+    Actually register new account, trying repeatedly if there are email
```

Comment:
nit: I think more info should be added to this docstring. We could document the `acme` and `config` parameters as well as the return value. The return value may be the same as `acme.register`, but why not document `:returns:` and `:rtype:` here so developers don't have to go searching through our code.